src/components/screens/GuideScreen/Hero.js

```diff
@@ -161,6 +160,37 @@ const GuideImage = styled.img`
     `}
 `;
 
+const BreadcrumbLink = styled(GatsbyLink)`
+  font-size: ${typography.size.s3}px;
+  line-height: ${typography.size.m1}px;
+  font-weight: ${typography.weight.bold};
+  margin-top: 16px;
+  margin-bottom: 20px;
+  color: ${color.lightest};
+
+  &:hover,
+  &:focus,
+  &:active {
+    color: ${color.lightest};
+  }
+
+  @media (min-width: ${breakpoint * 1.333}px) {
+    margin-top: 0;
+    font-size: ${typography.size.m1}px;
```

Comment:
```suggestion
    margin-left: 0;
```

Reset margin-left from mobile.
Preserve the 16px font-size on desktop too.